src/js/components/List/List.js

```diff
@@ -573,6 +672,7 @@ const List = React.forwardRef(
                     background={adjustedBackground}
                     border={adjustedBorder}
                     isDisabled={isDisabled}
+                    isPinned={isPinned}
```

Comment:
I don't think this is doing anything.

Reply:
You're correct, I'll make sure to remove that.